collector.go

```diff
@@ -163,7 +165,14 @@ func (c *collector) collectBatch(ch chan<- prometheus.Metric, metrics []types.Me
 	nr := len(results)
 	nm := len(metrics)
 	if nr != nm {
-		panic(fmt.Sprintf("not same length: %d != %d", nr, nm))
+		namespace := "unknown"
+		if nm > 0 {
+			namespace = *metrics[0].Namespace
+		}
+		level.Error(c.logger).Log("msg", "not same length", "results", nr, "metrics", nm, "namespace", namespace)
```

Comment:
The logger should already include the namespace and metricName: https://github.com/discordianfish/cloudwatch-exporter/blob/main/handler.go#L102
So `level.Error(c.logger).Log("msg", "not same length", "results", nr, "metrics", nm)` should be sufficient.

Reply:
@discordianfish fixed